src/js/actions/ui.js

```diff
@@ -49,6 +80,28 @@ define(function (require, exports) {
     ];
 
     /**
+     * Globally enable tooltips.
+     *
+     * @return {Promise}
+     */
+    var enableTooltips = function () {
+        return adapter.setPropertyValue(TOOLTIP_TIME_KEY, DEFAULT_TOOLTIP_TIME);
+    };
+    enableTooltips.writes = [locks.PS_APP];
+
+    /**
+     * Globally disable tooltips and clear any current tooltip.
+     *
+     * @return {Promise}
+     */
+    var disableTooltips = function () {
+        return adapter.setPropertyValue(TOOLTIP_TIME_KEY, DISABLED_TOOLTIP_TIME).then(function () {
+            adapterOS.setTooltip("");
+        });
+    };
+    enableTooltips.writes = [locks.PS_APP];
```

Comment:
I wish there were a better way to do this so that it would be harder to make this mistake.